src/plan_bot.py

```diff
@@ -258,9 +222,22 @@ def process_post(
         posts_db.document(post.id).set(post_record)
 
     operations_map = {
```

Comment:
I thought that adding the ability to provide arguments in the operations map gave us some useful flexibility (already exploited here) without too much added complexity.

Reply:
I do like the flexibility, but I don't think we need the separation of setting them up and recombining them! See comment below :)